addons/l10n_za/data/template/account.account-za.csv

```diff
@@ -100,9 +105,10 @@
 "610410","610410","Insurance","expense","False"
 "610420","610420","Travel and subsistence","expense","False"
 "610430","610430","Corporation tax expense","expense","False"
-"610440","610440","Foreign Exchange Gains/Losses","expense","False"
+"610440","610440","Depreciation","expense","False"
```

Comment:
I'm hesitant to replace an existing code with a completely different account. Use another code for this one, something similar to the other depreciation accounts 620...